example/README.md

```diff
@@ -0,0 +1,16 @@
+Themable Example
```

Comment:
Maybe better to use md headings (hashtag in front). E.g.:

# Themable example
## How to run the example
1. Be sure to run `npm run build:prod` in the library folder beforehand, since it will be linked as a module to this package.
2. Install the modules used by this app:
```
npm install
```

3. Link it to the library folder. Be sure that the lib folder contains the latest build bundled in the `./dist` folder:
```
npm run link:nice2
```
4. Start the project
```
npm start
```